### ISSUE TITLE
Visualize parameters loaded to a global ensemble by ert3

### ISSUE THREAD

**Issue by DanSava** [equinor/webviz-ert#203]
**Issue**
Ert3 supports the option of loading records directly to the database using the command:

`ert3 record load [record_name] [record/path/file_name.json]`

When this option is used a record is added to the "global" ensemble "experiment/path.__ensemble_records__ " but there is no connection in the database that this record from the global ensemble is connected to a particular ensemble that will be running in the future. Hence webviz-ert does not know that the added record ([record_name] in this example) is part of the global ensemble and not part of the run ensemble. 

*Scope*
Alow visualization of preloaded parameter records

**Comment by mortalisk** [comment]
Is it supposed to be connected to a future ensemble?

**Comment by DanSava** [comment]
Yes it should be connected somehow to a future ensemble or a future ensemble should know about it (discover it somehow)

**Comment by mortalisk** [comment]
If I understand correctly, we need to have some way of indicating that this parameter set is not currently supposed to appear as a parameter for the existing ensemble. But is it then supposed to be visible in another way?

**Comment by markusdregi** [comment]
This issue should be resolved as part of https://github.com/equinor/ert/issues/2707
